helm/flink-kubernetes-operator/templates/flink-operator.yaml

```diff
@@ -130,6 +141,10 @@ spec:
             - name: flink-artifacts-volume
               mountPath: /opt/flink/artifacts
             {{- end }}
+          {{- if .Values.tls.create }}
+            - name: flink-operator-cert-secret
+              mountPath: /opt/flink/artifacts/operator-cert
```

Comment:
AFAIK when there is an overlap in the mounts then one of them is going to be hidden (Either `/opt/flink/artifacts` or `/opt/flink/artifacts/operator-cert`). Maybe `/opt/flink/certs` would be a better choice.